platform/core/src/measurements/classes/MeasurementApi.js

```diff
@@ -203,13 +204,14 @@ export default class MeasurementApi {
 
   retrieveMeasurements(patientId, timepointIds) {
     const retrievalFn = configuration.dataExchange.retrieve;
+    const { server } = configuration;
```

Comment:
I think this is a valid solution, but we need to indicate that we expect `server` to be defined.

Here: https://github.com/OHIF/Viewers/blob/f5657595a59a996a984ea5fc5ce54d21148ddc3c/platform/core/src/measurements/classes/MeasurementApi.js#L18-L20

Here: https://github.com/OHIF/Viewers/blob/f5657595a59a996a984ea5fc5ce54d21148ddc3c/platform/core/src/measurements/classes/MeasurementApi.js#L11-L13

The easiest way to do this is by:

1. Not spreading props
2. Adding JSDoc
3. Potentially warning or throwing if `server` is not set

You can skip number three if `server` is required to instantiate the `MeasurementApi`, and you have checks in the constructor.

Reply:
That also means the shape of `server` should be defined/documented.

Reply:
Still need the shape of `server` and the `configuration` object defined somewhere.